newIDE/app/src/GameDashboard/GameCard.js

```diff
@@ -33,68 +49,105 @@ export const GameCard = ({
     if (!url) return;
     Window.openExternalURL(url);
   };
-
+  // TODO center the placeholder text
   return (
     <I18n>
       {({ i18n }) => (
         <Card key={game.id}>
-          <CardHeader
-            title={game.gameName}
-            subheader={
-              <Line alignItems="center" noMargin>
-                <Trans>Created on {i18n.date(game.createdAt * 1000)}</Trans>
-                {isCurrentGame && (
-                  <>
-                    <Spacer />
-                    <Chip
-                      size="small"
-                      label={<Trans>Currently edited</Trans>}
-                      color="primary"
-                    />
-                  </>
-                )}
-                {game.publicWebBuildId && (
-                  <>
-                    <Spacer />
-                    <Chip
-                      size="small"
-                      label={<Trans>Published on Liluo</Trans>}
-                    />
-                  </>
-                )}
-              </Line>
-            }
-          />
-          <CardActions>
-            <ResponsiveLineStackLayout
-              expand
-              noMargin
-              justifyContent="flex-end"
-            >
-              {game.publicWebBuildId && (
-                <RaisedButton
-                  label={<Trans>Open</Trans>}
-                  onClick={openGameUrl}
-                  primary
-                />
-              )}
-              <FlatButton
-                icon={<TuneIcon />}
-                label={<Trans>Details</Trans>}
-                onClick={onOpenDetails}
-              />
-              <FlatButton
-                icon={<PlaylistPlayIcon />}
-                label={<Trans>Builds</Trans>}
-                onClick={onOpenBuilds}
+          <Line>
+            {game.thumbnailUrl ? (
+              <img
+                src={game.thumbnailUrl}
+                style={{
+                  ...styles.image,
+                  width: 240,
+                  height: 135,
+                  minHeight: 135,
+                }}
+                alt={game.gameName}
+                title={game.gameName}
               />
-              <FlatButton
-                icon={<TimelineIcon />}
-                label={<Trans>Analytics</Trans>}
-                onClick={onOpenAnalytics}
+            ) : (
+              <Paper
+                variant="outlined"
+                style={{
+                  width: '240px',
+                  height: '135px',
+                  whiteSpace: 'normal',
```

Comment:
```suggestion
                  whiteSpace: 'normal',
                  display: 'flex',
```

(see other comment for explanation)